blueprints/module-unification-addon/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const stringUtil = require('ember-cli-string-utils');
+const date = new Date();
+
+module.exports = {
+  description: 'Generates an Ember addon with a module unification layout.',
+
+  locals(options) {
+    let entity = options.entity;
+    let rawName = entity.name;
+    let name = stringUtil.dasherize(rawName);
+    let namespace = stringUtil.classify(rawName);
+    let addonName = stringUtil.dasherize(rawName);
+
+    return {
+      name,
+      modulePrefix: name,
+      namespace,
+      addonName,
+      // addonModulePrefix: addonName,
```

Comment:
It's not clear to me the difference between `addonModulePrefix` and `addonName` and which one should I use.

Reply:
I don't see `addonModulePrefix` used anywhere in these files.

Reply:
It is not, I decided to use addon name. But I wanted to understand if there was a good reason for that weird naming